### ISSUE TITLE
Make Multiple Books Appear Side-by-Side in Story Selection

### ISSUE THREAD

**Issue by jvillemare** [BITNULLS/storybook#485]
**Due Date:** Sprint S6

**Description:**
If there are multiple books, they appear above each other

<img width="815" alt="image" src="https://user-images.githubusercontent.com/6162516/167420273-5c97db5c-cc9c-4fbd-942f-9cf88c0a27a2.png">

which is not great for user experience.

Convert the book card item template to something like this

<img width="551" alt="image" src="https://user-images.githubusercontent.com/6162516/167420752-0091e3f0-c90d-4ead-b5cd-b5f209f21762.png">

Basically: Make the book item card appear as a row, and so that they nicely stack onto of each other.

**Objectives:**
 - [ ] Modify the book item template into the description above (**keep the elements, sizing, fonts, and everything the same. Just rearrange the card and make it take up the full width**)
 - [ ] Take a screenshot of the new design
 - [ ] Make a PR
